remotes/docker/registry_test.go

```diff
@@ -52,8 +52,10 @@ func TestMatchLocalhost(t *testing.T) {
 		match bool
 	}{
 		{"", false},
-		{"127.1.1.1", false},
+		{"127.1.1.1", true},
 		{"127.0.0.1", true},
+		{"127.256.0.1", false},
```

Comment:
No such IP can exist

Reply:
Yes I'm aware, maybe I misunderstood the way the tests are meant to be written, my intention was to test that given such an IP beginning from `127` (since the current implementation checks based on `127`), if an invalid loopback is given, it returns `false`.

I will remove this test, thanks!

Reply:
You can keep test for 127.256.0.1 (with comment line) to ensure the function does not panic for invalid input string.

Reply:
(Btw, the current code seems to cause panic for "127-foobar.example.com")

Reply:
Oh, I see. Will look into it 👍🏻